src/gui/layout/qgslayoutitemguiregistry.h

```diff
@@ -413,6 +413,16 @@ class GUI_EXPORT QgsLayoutItemGuiRegistry : public QObject
      */
     bool addLayoutItemGuiMetadata( QgsLayoutItemAbstractGuiMetadata *metadata SIP_TRANSFER );
 
+    /**
+     * Unregisters the gui metadata for a layout item type.
+     */
+    bool removeLayoutItemGuiMetadata( int type );
+
+    /**
+     * Unregisters the gui metadata for a layout item type.
```

Comment:
```suggestion
     * Unregisters the GUI metadata for a layout item type.
     *
     * The \a metadata object will be deleted and should not be used after this call.
     *
     * \since QGIS 4.0
```